.github/workflows/build.yml

```diff
@@ -33,18 +33,25 @@ jobs:
         include:
           - target: x86_64-unknown-linux-musl
             os: ubuntu-latest
+            cross: true
           - target: aarch64-unknown-linux-musl
-            os: ubuntu-latest-arm-4core
+            os: ubuntu-latest
+            cross: true
           - target: x86_64-unknown-linux-gnu
             os: ubuntu-latest
+            cross: true
           - target: aarch64-unknown-linux-gnu
-            os: ubuntu-latest-arm-4core
+            os: ubuntu-latest
+            cross: true
           - target: x86_64-pc-windows-msvc
             os: windows-latest
+            cross: false
           - target: aarch64-apple-darwin
             os: macos-latest
+            cross: false
           - target: x86_64-apple-darwin
             os: macos-13
```

Comment:
at some point we need to use cross-compilation for osx-64 as well once github drops support for that... but maybe we can just stop supporting macos-64 at that point

Reply:
Cross does not yet have official arm64 images, but there are some community builds we could use once GitHub drops their runner support.